### DISCUSSION TITLE
How to make a dark mode toggle that by default prefers color scheme?

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/tailwindcss#8184] [Help]
So in the docs it says that we can use the `class` mode if we want to toggle between light and dark modes. But the problem with this approach is that it is not possible to by default respect OS preferences and `prefers-color-scheme`. How does one go about creating a 3-way dark mode toggle of `auto`, `light` and `dark`?

**Comment by adamwathan** [comment]
Hey! Check the end of the docs page:

https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually

<img width="841" alt="image" src="https://user-images.githubusercontent.com/4323180/164953088-55eb45d0-ca5b-4d64-89d5-ed4e4d686c2e.png">
